embedded/ee10-servlet-server/src/test/java/examples/EmbedMeTest.java

```diff
@@ -29,33 +29,45 @@
 import static org.hamcrest.Matchers.containsString;
 import static org.hamcrest.Matchers.is;
 
-public class EmbedMeTest
+class EmbedMeTest
 {
     private Server server;
 
     @BeforeEach
-    public void startServer() throws Exception
+    void startServer() throws Exception
     {
         server = EmbedMe.newServer(0);
         server.start();
     }
 
     @AfterEach
-    public void stopServer()
+    void stopServer()
     {
         LifeCycle.stop(server);
     }
 
     @Test
-    public void testGetWelcome() throws IOException, InterruptedException
+    void testGetWelcome() throws IOException, InterruptedException
     {
-        HttpClient client = HttpClient.newBuilder().version(HttpClient.Version.HTTP_1_1).build();
-        HttpRequest request = HttpRequest.newBuilder()
+        var client = HttpClient.newBuilder().version(HttpClient.Version.HTTP_1_1).build();
```

Comment:
No, keep the variable, don't fall back to `var`.
The use of `var` makes for bad and hard to follow examples. (the whole point of this repo)